### PR TITLE
Fix bugs - constant and error throwing

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Fix timeout constant for waiting on workspace startup. Fix error throwing when waiting for a workspace to be up and running times out.

### What issues does this PR fix or reference?
Currently, when waiting for the workspace to be running & switched to IDE fails, you can see in the logs that it passed:

```
Validation of workspace start
          ▼ DriverHelper.navigateToUrl https://che-eclipse-che.10.0.101.167.nip.io/dashboard/#/ide/admin/petclinic-dev-environment
    ✓ Open workspace (92ms)
          ▼ Ide.waitAndSwitchToIdeFrame
          ▼ Ide.waitIde
          ▼ ProjectTree.openProjectTreeContainer
          ▼ Ide.waitLeftToolbarButton
    1) Wait for workspace readiness
```

### How to test this PR?
Run Happy path or some devfile test.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by dmytro-ndp — APPROVED** [review]
Looks good.

**Comment by Katka92** [comment]
[crw-ci-test]

**Review by ScrewTSW — CHANGES_REQUESTED** [review]

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 1]
```diff
@@ -49,6 +49,8 @@ export class Ide {
                 Logger.warn('StaleElementException occured during waiting for IDE. Sleeping for 2 secs and retrying.');
                 this.driverHelper.wait(2000);
                 await this.driverHelper.waitAndSwitchToFrame(By.css(Ide.IDE_IFRAME_CSS), timeout);
+            } else {
+                throw err;
```
[the remark this answers is not in the document]
Comment (Katka92) [reply tests/e2e/pageobjects/ide/Ide.ts:54]: Hm, good point, will add.

File: tests/e2e/testsLibrary/ProjectAndFileTests.ts [hunk 2]
```diff
@@ -21,7 +22,7 @@ const editor: Editor = e2eContainer.get(CLASSES.Editor);
 export function waitWorkspaceReadiness(sampleName : string, folder: string) {
     test('Wait for workspace readiness', async () => {
         await ide.waitAndSwitchToIdeFrame();
-        await ide.waitIde();
+        await ide.waitIde(TimeoutConstants.TS_SELENIUM_START_WORKSPACE_TIMEOUT);
```
[the remark this answers is not in the document]
Comment (Katka92) [reply tests/e2e/testsLibrary/ProjectAndFileTests.ts:25]: Yes, but in this case, this method is used to wait for a workspace to spin up AND load ide, so we need to have TS_SELENIUM_START_WORKSPACE_TIMEOUT there.

[the remark this answers is not in the document]
Comment (ScrewTSW) [reply tests/e2e/testsLibrary/ProjectAndFileTests.ts:25]: I see

**Review by ScrewTSW — APPROVED** [review]
LGTM

**Pull request merged**